de4a-connector/src/main/java/eu/de4a/connector/service/spring/Conf.java

```diff
@@ -349,6 +361,15 @@ public MessageSource messageSource() {
 		messageSource.setDefaultEncoding(StandardCharsets.UTF_8.name());
 		return messageSource;
 	}
+	
+	@PostConstruct
+	public void kafkaSettings() {
```

Comment:
Good. You also need to add the HttpClientSettings for the KafkaClient if the HTTP mode is enabled.
[A not complete example from the Connector-mock](https://github.com/de4a-wp5/de4a-connector-mock/commit/04c8b99f2da44eb6765cc3c0a73ce7916723977c#diff-12440bbb9f280db64476961318b82b3b4146be64c6f242a17578235cae09ef0aR31)
Just take the http.proxy stuff configured and add it to the HttpClientSettings Object and the call
DE4AKafkaSettings.setHttpClientSettings() with that object